### DISCUSSION TITLE
Builds timing out at 60 minute mark

### DISCUSSION THREAD

**Discussion by ghost** [actions/actions-runner-controller#2757] [Questions]
Hi there!  I have a very barebones configuration for my RunnerDeployment and ARC in general, and am finding that our builds (which are long running unity builds) are timing out at the 60 minute mark consistently.  Is there a parameter that will let me increase the allowable time for an action to run?  Feeling like there is something defaulted to 60 minutes but I can't find it in the documentation..

**Comment by cniessigma** [comment]
Is [timeout-minutes](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idtimeout-minutes) set in your workflow job definition? That's actually what determines the workflow timeout rather than anything in ARC.

**Reply by Fitzbert-Fitz** [reply]
I see the same issue: timeout-minutes is set to 720 minutes on job level.
